apps/web/src/design-system/table/Table.tsx

```diff
@@ -109,10 +109,12 @@ export function Table({
 
       <MantineTable className={classes.root} {...defaultDesign} {...getTableProps()} {...props}>
         <thead>
-          {headerGroups.map((headerGroup) => (
-            <tr {...headerGroup.getHeaderGroupProps()}>
+          {headerGroups.map((headerGroup, i) => (
+            <tr key={i} {...headerGroup.getHeaderGroupProps()}>
```

Comment:
Index as a key is a bad practice, we should probably have the `eslint` rule for that.
But from what I do see is that the `react-table` is auto-generating the keys for the header groups, so we don't need to do that.

Reply:
Fixed